media/css/pebbles/includes/mixins/_at2x.scss

```diff
@@ -0,0 +1,56 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+// A helper mixin for applying high-resolution background images (based on http://www.retinajs.com)
+// File-names should follow:
+//  example.png
+//  example-high-res.png
+// Usage:
+//  @include at2x('/media/img/foo.png', 100px, 100px);
+
+@mixin at2x($path, $w: auto, $h: auto) {
+    $hr-suffix: '-high-res';
+
+    // Set a counter and get the length of the image path.
+    $position: -1;
+    $strpath: '#{$path}';
+    $length: str-length($strpath);
+
+    // Loop through the image path and find the position of
+    // the dot to determine the file extension.
+    @for $i from $length through $length - 10 {
```

Comment:
To be honest, I'm not sure what it does either. I took this code from retinajs and the "from x through z" syntax is unfamiliar to me, seems it's a Sass thing. According to [their docs](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#_10) I think the `$length - 10` is just a way to ensure the end point is always a lower number than the start point so the loop will always increment. Am I reading that right? I will happily add an explanatory comment once we figure out how to explain it :)